scripts/secrets_manager

```diff
@@ -1,39 +1,47 @@
 #!/bin/bash
 
 function check_dependencies() {
+  if [[ $1 == "-exit_on_install" ]]; then
+    EXIT_ON_INSTALL="YES"
+  fi
+
   echo '----------------------------'
   echo 'Checking dependencies'
   echo '----------------------------'
 
+  DEPENDENCY_INSTALLED="NO"
   if ! [ -x "$(command -v age)" ]; then
-  	echo 'age is not installed. installing with `brew install age`' >&2
-  	brew install age;
+    echo 'age is not installed. installing with `brew install age`' >&2
+    brew install age;
 
     if ! [ -x "$(command -v age)" ]; then
-  	  echo 'age is still not installed. installing with `brew install age`' >&2
-  	  exit 1
-	  else 
-      echo 'all dependencies installed' 
+      echo 'age is still not installed. installing with `brew install age`' >&2
+      exit 1
+    else
+      DEPENDENCY_INSTALLED="YES"
     fi
-	else 
-    echo 'all dependencies installed' 
-	fi
+  fi
 
-}
+  if ! [ -x "$(command -v rename)" ]; then
+    echo 'rename is not installed. installing with `brew install rename`' >&2
+    brew install rename;
```

Comment:
good idea to just add this here - I can remove it from the notion now.

Reply:
ok sounds good